NEWS.md

```diff
@@ -54,6 +54,9 @@
 
 * `test_file()` gains a `desc` argument which allows you to run a single 
   test from a file (#1776).
+  
+* `JunitReporter()` now uses ensures numeric values are saved the xml file 
```

Comment:
Can you please put this under the development version heading?

Reply:
I'm sorry, my bad, didn't spot that there are 2 versions back to back. Want me to copy the same subsection structure (## Lifecycle changes, ## New features etc.) or just put it under the development version and that's it?

Reply:
No, just copy the bullet. We add the subsections as part of the release process.

Reply:
Gotcha, done :)